_posts/2016/2016-07-26-xconf-2016.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title: "XConf 2016 Hamburg"
+date: "2016-07-26 00:00:00"
+image: blog-header/lego.jpeg
+categories: events
+authors: ["Oleksandr V", "Renato"]
+---
+
+Not so long time ago, in one of so few sunny days in Hamburg I have had an opportunity to visit
+[XConf](https://info.thoughtworks.com/Xconf-hamburg-2016.html). Here in ePages, developers have a great support by the company if they are interested in self-education.
+
+This yearly event took place on 12th of July in a cozy place [Betahaus](http://hamburg.betahaus.de/) in format of one day conference covering "broad range of technology topics".
+
+Thanks to the great job made by [ThoughtWorks](https://www.thoughtworks.com/) I have enjoyed time participating it.
+
+It was a surprise for me to meet another developer from my company deciding to visit this conference as well.
+
+{% image blog/blog-twitter-xconf-2.png %}
+
+## Keynote
+
+[James Lewis](https://twitter.com/boicy) presented the keynote talking about Lean Software Engineering, [LEGO](http://lego.com) and [Star Wars](http://www.starwars.com/).
+Sometimes when people present processes they make theirs presentation too abstract and might loose the audience on the way.
+What was interesting in this presentation, that he was able to illustrate in a form of a story, processes problems and discussed ways of solving them.
+And everything with help of [LEGO](http://lego.com) building a Millennium Falcon.
+With such presentation techniques he was able to explain several concepts including cycle-time and throughput, continuous process improvement to people who are not familiar with the topic.
+
+## Presentations
+
+Every participant could choose one of the two tracks running in parallel.
+
+We have decided to visit Track 1 because there were lots of topics we want to get more insides on how other companies are dealing with.
+
+**Content of Track 1:**
+
+ - Containerizing a web shop - Building the microservice platform for otto.de
+ - Implementing Infrastructure as Code
+ - A High-Performance Solution to Microservice UI Composition
+ - Serverless Architecture - Tales from a world without servers
+ - Alert overload: How to adopt a MS architecture without being...
+
+## Impression
+
+In general, presentations were really nice: well structured content, experienced speakers and interesting topics.
+One good takeaway of the conference was to see that many companies are using and applying the same techniques and tools that we are using internally.
+
+Especially, it was really interesting to hear about experience of introducing microservices architecture on such giant environment as [otto.de](http://otto.de)
+
+It is good to now that one of the leaders of German e-Commerce market following the same principles as we are following in our team.
```

Comment:
It is good to know that one of the leaders of the German e-commerce market follows the same principles as we are following in our team.